src/app/developer/Developer.tsx

```diff
@@ -13,7 +13,8 @@ interface State {
 export default class Developer extends React.Component<{}, State> {
   constructor(props) {
     super(props);
-    const urlParams = parse(window.location.href);
+    // we ask typescript to trust that we won't do array values as URL params
+    const urlParams = parse(window.location.href) as NodeJS.Dict<string>;
```

Comment:
not a failing of this change, the same problem happens if you bother to install `simple-query-string`'s types package